docs/Versioning.md

```diff
@@ -53,13 +66,33 @@ As a general principle, implementations SHOULD be robust in the face of missing
 
 By way of example, the `ModelProto.ir_version` MUST be present in every model.  The ONNX checker (`onnx/checker.py`) will enforce these rules.
 
-ISSUE: decide and document how we want to handle changes to the format that are forward compatible but will cause the generated code to introduce a breaking change (e.g., changing the data type of a field from int64 to int32, int64 to double or bytes to string).  I would recommend that we never do it after we hit 1.0.0 - prior to that we should be fairly liberal with changes to size (e.g., `int32<>int16`, `float<>double`) and conservative with changes to value space (e.g., `integral<>floatingpoint`, `string<>int`, `scalar<>message`)
+Because onnx.proto is expected to be consumed by multiple independent developers, changes to onnx.oroto SHOULD NOT break code that depends on generated language bindings (e.g., changing the type of an existing field).
 
 ISSUE: define type compatiblity rules either here or under model versioning - probably here
 
 ## Operator versioning
 
-ISSUE: sort out the design and then document.
+ONNX is defined such that the IR can evolve independently from the set of operators. In ONNX, operators represent both the signature and semantics of a given operation.  Operators are abstract interfaces in that they do not imply a specific implementation; rather, they are simply the contract between a model author and the implementations that model may execute on. 
+
+A given operator is identified by a three-tuple: (domain, op_type, op_version), written domain.op_type:op_version in prose (e.g., com.acme.FastConv:3).  Nodes in graphs always refer
```

Comment:
So, going along with "expunge op version", this should be `(domain, opset_version, op_type)`. It might also be less ambiguous if we say `opset_domain` rather than `domain` but I'm OK with the shorter name.

Reply:
See comment above. We definitely need a different name. Look for a checkin in 30 min.

Reply:
how about calling it domain_version?